cmd/sync-rover-groups/README.md

```diff
@@ -0,0 +1,22 @@
+# sync-rover-groups
+
+## What it does
+
+`sync-rover-groups` is a tool to resolve the groups in [the manifests](https://github.com/openshift/release/tree/master/clusters) of CI clusters
+in the release repo.
+
+
+## Why it exists
+
+Only [rover-groups](https://rover.redhat.com/groups/) are allowed in the manifests.
+`sync-rover-groups` collects the groups in the manifests and resolves their members
+and save resolved groups in a file.
+
+## How it works
+
+`sync-rover-groups` resolve the group by querying the Red Hat LDAP server.
+
+## How is it deployed
+
+The cronjob [sync-rover-groups-update](https://console-openshift-console.apps.ocp-c1.prod.psi.redhat.com/k8s/ns/ocp-test-platform/batch~v1~CronJob/sync-rover-groups-update) ([definition](http://TODO))
```

Comment:
The URL will be updated after https://github.com/openshift/release/pull/25357 is merged.

Reply:
This ties into my previous comment, I don't think we need to provide exact lines in a file that changes often.

Reply:
Updated.